### PR TITLE
Karma code coverage reports.

### PR DESCRIPTION
**Pull request by afenton90**
Changed Karma configuration so code coverage reports are generated each time tests are run using the `bj test` command. Coverage reports are done after the source code has been transpiled, meaning that the reports actually look at the generated ES5 code.
The version of enzyme has also been upgraded to include the latest features.

### CODE REVIEW

**Comment by joshnesbitt** [comment]
:shipit:

File: package.json [hunk 1]
```diff
@@ -62,7 +63,6 @@
     "karma-mocha": "^1.0.1",
     "karma-phantomjs-launcher": "^1.0.0",
     "karma-sinon-chai": "^1.2.0",
-    "karma-sourcemap-loader": "0.3.7",
```
Comment (mackstar) [inline package.json]: I am guessing this was intentional?

**Comment by mackstar** [comment]
Thanks for the contribution @afenton90 good stuff!

File: package.json (hunk 1 above)
Comment (afenton90) [inline package.json]: Yeah this was no longer needed in the Karma configuration.

**Pull request merged**